apps/wing/src/analytics/disclaimer.ts

```diff
@@ -1,5 +1,9 @@
 import chalk from "chalk";
-import { AnalyticsStorage } from "./storage";
+import { AnalyticsConfig, AnalyticsStorage } from "./storage";
+
+// simplest way I could think of to allow us to update the disclaimer
+// and force it to be displayed again
```

Comment:
You're making it sound like this is a crazy thing to do haha

Versioning the disclaimer is a great idea.

Reply:
Oh yea I agree, 😅 I was thinking more its just a hardcoded string we need to manually update.